Automattic/jetpack
# Subscriptions Block: Increase the priority

This increases the priority of the subscriptions block.

We are using this block in a block pattern in the Quadrat theme. We need to check that the block is loaded before we load the pattern, which means that we have to wait for this action to fire. The core block patterns are also hooked to the `init` action, so they end up appearing in the list earlier than the theme patterns, which isn't ideal. By loading this block with an earlier priority we are able to load the theme patterns earlier than the core ones.





#### Changes proposed in this Pull Request:

* Increase the priority of the subscriptions block

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:






To test use this PR: https://github.com/Automattic/themes/pull/3774
Check that the core block patterns load after "Quadrat".

Changelog:
Incease the priority for the Subscriptions block
